docs/docs/general/03-Element Selection/recommended-practices.md

```diff
@@ -0,0 +1,171 @@
+---
```

Comment:
Only looked over this page roughly. I think this page is worth talking about synchronously.